Simplenote/src/main/java/com/automattic/simplenote/utils/DateTimeUtils.java

```diff
@@ -37,4 +38,12 @@ public static String getDateTextString(Context context, Calendar calendar) {
         );
         return new SimpleDateFormat(pattern, Locale.getDefault()).format(calendar.getTime());
     }
+
+    public static Calendar getDateCalendar(String json) throws ParseException {
+        String pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
```

Comment:
this looks like an ISO8601 string. I see `parseOffsetISO8601` in SDK 24 which I guess means we can't use that here. I looked briefly through `androidx` and didn't see one. do you know if there's an existing ISO8601 parser available in one of the compatibility or other libraries we already use?

Reply:
Ok, I'll look for solutions!